pkg/storage/snapshot/source.go

```diff
@@ -172,6 +238,25 @@ func (s *vmSnapshotSource) Unlock() (bool, error) {
 	return true, nil
 }
 
+func (s *vmSnapshotSource) verifyVolumesBound(pvcNames []string) (bool, error) {
+	for _, pvcName := range pvcNames {
+		obj, exists, err := s.controller.PVCInformer.GetStore().GetByKey(cacheKeyFunc(s.vm.Namespace, pvcName))
+		if err != nil {
+			return false, err
+		}
+		if !exists {
+			return false, k8serrors.NewNotFound(appsv1.Resource("persistentvolumeclaim"), pvcName)
+		}
+
+		pvc := obj.(*corev1.PersistentVolumeClaim).DeepCopy()
+		if pvc.Status.Phase != corev1.ClaimBound {
```

Comment:
Not a problem with populators but it turns out that we have at least one storage partner that is CSI, supports snapshots, but is forced to do "usePopulator: false" annotation for "reasons" so I'm wondering if it would make sense to do the equivalent of https://github.com/ShellyKa13/kubevirt/blob/vmsnapshot-unbound-volumes/vendor/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/utils.go#L28-L41